content/recipes/introduction/FAIR-and-knowledge-graphs.md

```diff
@@ -431,67 +396,7 @@ In the previous 2 sections, we have highlighted 2 of the most successful project
 Property Graphs as data structure for storing information and building knowledge graphs. 
 However, none of these implementations are officially an approved W3C standards. While it does not really matter
 at this stage, it was realized that such fragmentation was hurting and in 2019, the `open Graph Query Language` (oGQL) 
-working group released an initial specification
-
-
-## How are knowledge graphs generated?
-
-{footcite}`pmid32637040`
-
-
-### Semantic models and ontologies: when knowledge engineers step in.
```

Comment:
I personally prefer having this section in a separate recipe.

Reply:
not exclusive. we can keep the section but also expand on a specific recipe. we have several examples, including one from the fellows.